src/bump-package-version/index.ts

```diff
@@ -1,10 +1,23 @@
 import {DangerDSLType} from 'danger'
 
+/**
+ * Checks all @includePaths and in case of presence of edited files 
+ * requires version in `package.json` to be updated.
+ * Parameter @restrictToBranches defines branches to run the check for.
+ * If the parameter is empty the branch is not validated.
+ */
 export const bumpPackageVersion = async (params: {
   danger: DangerDSLType
   fail: (message: string) => void
   includePaths: Array<string>
+  restrictToBranches?: Array<string>
 }) => {
+  if(!params.restrictToBranches ||
+    !params.restrictToBranches.includes(params.danger.github.pr.base.ref)
```

Comment:
Use `!params.restrictToBranches?.includes(params.danger.github.pr.base.ref` instead of `!params.restrictToBranches || !params.restrictToBranches.includes(params.danger.github.pr.base.ref`.